app/models/job.rb

```diff
@@ -62,7 +62,13 @@ def app_inspect(bin_count = 5, job_name = 'ondemand/sys/dashboard/sys/bc_osc_jup
         property_hash[job.tres.scan(match_str).flatten.map(&:to_i).sum] += 1
       end
       max_property = property_hash.keys.max + 1
-      bin_size = (max_property.to_f / bin_count).ceil
+      # When slider is all the way to the right, use a bin_size of 1 and set bin_count to max
+      if bin_count == 21
```

Comment:
21 seems to be a special number. Where did we get that number from?

Reply:
The bin slider is set to choose a number of bins between 1 and 20. I increased the size of the slider to include 21 for the 1 bin/item setting.

Reply:
OK I see that now. Hmmmm, OK I thought it was more dynamic than that. I.e., you'd get the options for as many bins as you could.  Cool.

https://github.com/OSC/osc-reporting/blob/d87010d428e1e9e7df04253652e028d539846bf3/app/views/app_inspector/index.html.erb#L26